src/seedu/addressbook/AddressBook.java

```diff
@@ -1138,14 +1138,17 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
-     */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
+     * @return  string without the prefix
+     */
+    private static String removePrefix(String fullString, String prefix) {
+    	if (fullString.startsWith(prefix)) {
```

Comment:
When you modify the behavior, remember to update documentation and tests as well.